// [[Reason: TSC adds type reference directives.]] ////

//// [tests/cases/conformance/node/nodeModulesImportAssignments.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,56 +1,10 @@
 
 
 //// [file.d.ts]
-/// <reference path="types.d.ts" />
 export import fs2 = require("fs");
 
 //// [index.d.ts]
-/// <reference path="types.d.ts" />
 export import fs2 = require("fs");
 
 //// [subfolder/index.d.ts]
-/// <reference path="../types.d.ts" />
 export import fs2 = require("fs");
-
-/// [Errors] ////
-
-file.ts(6,1): error TS9024: Declaration emit for this expression requires adding a type reference directive to '.types.js' with --isolatedDeclarations.
-index.ts(4,1): error TS9024: Declaration emit for this expression requires adding a type reference directive to '.types.js' with --isolatedDeclarations.
-subfolder/index.ts(4,1): error TS9024: Declaration emit for this expression requires adding a type reference directive to './types' with --isolatedDeclarations.
-
-
-==== subfolder/index.ts (1 errors) ====
-    // cjs format file
-    import fs = require("fs");
-    fs.readFile;
-    export import fs2 = require("fs");
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS9024: Declaration emit for this expression requires adding a type reference directive to './types' with --isolatedDeclarations.
-==== index.ts (1 errors) ====
-    // esm format file
-    import fs = require("fs");
-    fs.readFile;
-    export import fs2 = require("fs");
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS9024: Declaration emit for this expression requires adding a type reference directive to '.types.js' with --isolatedDeclarations.
-==== file.ts (1 errors) ====
-    // esm format file
-    const __require = null;
-    const _createRequire = null;
-    import fs = require("fs");
-    fs.readFile;
-    export import fs2 = require("fs");
-    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS9024: Declaration emit for this expression requires adding a type reference directive to '.types.js' with --isolatedDeclarations.
-==== package.json (0 errors) ====
-    {
-        "name": "package",
-        "private": true,
-        "type": "module"
-    }
-==== subfolder/package.json (0 errors) ====
-    {
-        "type": "commonjs"
-    }
-==== types.d.ts (0 errors) ====
-    declare module "fs";
\ No newline at end of file
